apps/frontend/docs/styleguide.md

```diff
@@ -2,14 +2,14 @@
 
```

Comment:
Lovely documentation !

Reply:
I agree!

(It has been completed on other branches)